lib/modules/hosts/youtube.js

```diff
@@ -53,7 +53,7 @@ export default new Host('youtube', {
 		return {
 			type: 'IFRAME',
 			embed: src,
-			embedAutoplay: insertParams(src, { autoplay: 1 }),
+			embedAutoplay: insertParams(src, { autoplay: 1, rel: 0 }),
```

Comment:
This needs to be done on line 25; this will only affect videos with autoplay (unless yt only shows related videos when you enable autoplay? but I don't think that's the case)